iml-services/iml-ostpool/src/main.rs

```diff
@@ -31,7 +31,15 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
             let client = client.clone();
 
             async move {
-                let fsid = client.fsid(&fsname).await?;
+                // If fsid finds no match, assume FS hasn't been added to IML and skip
+                let fsid = match client.fsid(&fsname).await {
+                    Ok(id) => id,
+                    Err(Error::NotFound) => {
```

Comment:
nit. `Err(Error::NotFound)` seems like `None` to me, I wonder if this fn should return `Option<Result<.., ..>>` instead.